apps/drupal-default/particle_theme/templates/block/block--utc-highlights-block.html.twig

```diff
@@ -133,49 +148,54 @@
         <ul
           class="grid col-auto auto-cols-fr lg:grid-flow-col md:grid-rows-1 list-none gap-14 px-0 mt-12 mb-2 justify-items-center">
           {% for content in info_1.0['#icons'] if info_1.0['#icons'] is not empty %}
-            <li class="px-8 py-4 text-lg text-center w-full {{ description_color }}">
-              {% if info_1.0['#icons'][loop.index0]['#name']  != 'fa-' %}
-                <div class="mb-4 {{ icon_color }}">
-                  <span
-                    class="{{ info_1.0['#icons'][loop.index0]['#style'] }} fa-3x {{ info_1.0['#icons'][loop.index0]['#name'] }}"></span>
-                </div>
-              {% endif %}
-              <span
-                class="block pb-2 {{ info_1.0['#icons'][loop.index0]['#name'] != 'fa-' ? 'text-3xl' : 'text-4xl' }} font-bold {{ info_color }}">{{ info_1.0['#icons'][loop.index0]['#info'] }}</span>
-              {{ info_1.0['#icons'][loop.index0]['#description'] }}
-            </li>
+            {% if info_1.0['#icons'] is not empty  %}
```

Comment:
This seems redundant? Isn't that state already set in 150?

Reply:
What if you remove the if from 150? Does it still work?

Reply:
I will try it. I inherited this from Brittany.

Reply:
If you look at 168 it is a similar "for"/"if" so it might not make any difference but it's cleaner.